src/api.ts

```diff
@@ -0,0 +1,68 @@
+import express from 'express';
+import { voteReportWithStorage, rpcError, rpcSuccess } from './helpers/utils';
+import log from './helpers/log';
+import { queues } from './lib/queue';
+
+const router = express.Router();
+
+router.post('/votes/generate', async (req, res) => {
```

Comment:
Add a route similar to other servers https://github.com/snapshot-labs/snapshot-hub/blob/master/src/api.ts#L18-L26 so we can know if there is something is blocking a deployment.